FilmeApp/components/ExplorePage.js

```diff
@@ -35,6 +34,7 @@ export default class ExplorePage extends React.Component {
     .catch((error) => {
       console.log(error);
     });
+    BackHandler.addEventListener('hardwareBackPress', this.handleBackPress);
```

Comment:
Why in the constructor instead of didMount?
Use this.backHandler if it works.

Reply:
It's out of the constructor